docs/content/operations/web-consoles.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: doc_page
+title: "Web Consoles"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+# Web Consoles
+
+Druid has a few console UIs that provide cluster management functionality, described below.
```

Comment:
I think we should strongly emphasize the new console and direct people to use that. Because the new console calls the coordinator and overlord consoles legacy consoles I think the documentation should also reflect this.

Reply:
Moved these under a "Legacy" section, added a recommendation to use the new console if possible